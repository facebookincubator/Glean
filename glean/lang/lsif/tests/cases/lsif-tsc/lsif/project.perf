{ "@generated": null, "lsif.Project.1": 1 }