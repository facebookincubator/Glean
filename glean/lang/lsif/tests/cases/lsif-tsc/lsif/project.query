query: lsif.Project _
perf: True
transform: [gensort, []]
