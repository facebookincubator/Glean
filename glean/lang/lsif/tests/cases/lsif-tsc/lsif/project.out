[ "@generated", { "key": { "kind": 49 } } ]